snuba/web/rpc/__init__.py

```diff
@@ -117,6 +121,17 @@ def execute(self, in_msg: Tin) -> Tout:
             span.description = self.config_key()
         self.__before_execute(in_msg)
         error = None
+        if (
+            hasattr(in_msg, "meta")
+            and hasattr(in_msg.meta, "start_timestamp")
+            and hasattr(in_msg.meta, "end_timestamp")
+        ):
+            start = in_msg.meta.start_timestamp.ToDatetime()
+            end = in_msg.meta.end_timestamp.ToDatetime()
+            if (end - start).days > MAXIMUM_TIME_RANGE_IN_DAYS:
+                timestamp = Timestamp()
+                timestamp.FromDatetime(end - timedelta(days=MAXIMUM_TIME_RANGE_IN_DAYS))
```

Comment:
I have no opposition to this, but I'm just curious if there's a reason we're starting at the `end_timestamp` when we trim vs. the `start_timestamp`